bpaotu/bpaotu/static/bpaotu/table.js

```diff
@@ -520,9 +520,19 @@ $(document).ready(function() {
             success: function(result) {
                 authentication_token = result;
                 run_setup_functions();
+
+                var data_portion = $.parseJSON(authentication_token.split("||")[1]);
+                var username = (data_portion.email).split("@")[0];
+                var user_string = username + " | " + '<a href="/user/_logout">Sign Out</a>';
+
+                $("#user_id").html(user_string);
```

Comment:
don't do this - what if the username somehow has HTML in it? use jQuery to safely build DOM elements (check the jQuery docs)

Reply:
Fixed.